orderer/common/multichannel/registrar.go

```diff
@@ -239,6 +293,24 @@ func (r *Registrar) Initialize(consenters map[string]consensus.Consenter) {
 	}
 }
 
+// initialize the channel participation API file repos. This creates the
+// joinblock (and TODO removeblock) file repos for the orderer, creating
+// the fileRepoDir on the filesystem if it does not already exist.
+func (r *Registrar) initializeFileRepos() {
+	// Initialize channel-participation API file repos
+	fileRepoDir := filepath.Join(r.config.FileLedger.Location, "filerepo")
+	logger.Infof("Channel Participation API enabled, registrar initializing with file repo %s", fileRepoDir)
+
+	joinBlockFileRepo, err := filerepo.New(fileRepoDir, "joinblock")
+	if err != nil {
+		logger.Panicf("Error initializing join block file repo: %s", err)
+	}
+
+	r.joinBlockFileRepo = joinBlockFileRepo
+
+	// TODO removeblock file repo
+}
```

Comment:
This does not belong here... the registar only manages the join-block repo. The remove repo will be internal to the file-ledger.

Reply:
Removed